.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "Static Analysis"
+
+on:
+  push:
+    branches: [ "master", "next"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
+    - cron: '37 15 * * 1'
```

Comment:
Do we need a `cron` job here?

Reply:
This were their default settings... we probably don't need it?

Reply:
Since this will run on pushes to `master` and `next`, as well as on PRs to `master` (if I understand correctly) then the `cron` seems unnecessary.

Reply:
Addressed in [a44e275](https://github.com/mfem/mfem/pull/3162/commits/a44e27583c0cf09db5af96ffda9ec46c05562d19)